Source/voicesdk_unreal_demo/voicesdk_unreal_demo.Build.cs

```diff
@@ -0,0 +1,23 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+using UnrealBuildTool;
+
+public class voicesdk_unreal_demo : ModuleRules
+{
+	public voicesdk_unreal_demo(ReadOnlyTargetRules Target) : base(Target)
+	{
+		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
+	
+		PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore" });
+
+		PrivateDependencyModuleNames.AddRange(new string[] {  });
+
+		// Uncomment if you are using Slate UI
```

Comment:
minor, do we need these commented out parts?

Reply:
We do not. Those lines are added by the engine when a project is switched to a C++ project, which is not needed in this case. I have reverted this file back to it's original state.